pixel/pixel.go

```diff
@@ -202,6 +204,29 @@ func (c RGB444BE) RGBA() color.RGBA {
 	return color
 }
 
+type MonochromeVertical bool
+
+func NewMonochromeVertical(r, g, b uint8) MonochromeVertical {
+	return MonochromeVertical((r > 0) || (g > 0) || (b > 0))
+}
+
+func (c MonochromeVertical) BitsPerPixel() int {
+	return 1
+}
+
+func (c MonochromeVertical) RGBA() color.RGBA {
+	value := uint8(0)
+	if c {
+		value = 255
+	}
+	return color.RGBA{
+		R: value,
+		G: value,
+		B: value,
+		A: value,
```

Comment:
The alpha value is always 255 (displays generally aren't transparent).

```suggestion
		A: 255,
```

Reply:
Done.